### DISCUSSION TITLE
Allow renaming of industries in Scenario Editor

### DISCUSSION THREAD

**Discussion by kevinfields777** [OpenTTD/OpenTTD#10189] [Features]
More info on the issue found here: 
https://www.reddit.com/r/openttd/comments/z1hh83/how_to_assign_industries_to_a_certain_city/

This is definitely a QoL issue, but I think it would be beneficial to be able to rename an industry to something else, no matter which city the industry is closest to.

**Comment by EmperorJake** [comment]
Sounds like a good idea, but internally the industry will still be associated with the original town even if it's renamed. This might end up confusing people who try to serve it in a town growth script.

**Reply by LC-Zorg** [reply]
This is definitely not true. :) I don't know of a single script that would work this way. With the exception of the New World Disorder script, where production is counted, not deliveries, deliveries everywhere are counted according to the stations assigned to the city. So changing the name of the company will not change anything here.

**Comment by LC-Zorg** [comment]
This could be a useful option when creating scenarios based on a real map and real enterprises, or when the author wants to spice up the scenario by adding interesting names. An example of a "real" scenario can be the server Map of Poland, where the vast majority of enterprises on the map correspond to real enterprises. In the old version, inscriptions with names were placed above many enterprises. It would be nice to be able to name them in the enterprise window instead of on the game map. :)
